src/internal/constants/uiConfig.ts

```diff
@@ -540,3 +542,37 @@ export const DEFAULT_XIRR_UICONFIG: IUiConfig = {
     ...defaultRootUiConfigProperties,
     ...disabledExportConfig,
 };
+
+const geoMeasuresBase = {
+    ...measuresBase,
+    itemsLimit: DEFAULT_PUSHPIN_METRICS_COUNT,
+    allowsReordering: false,
+    isShowInPercentVisible: false,
+    canAddItems: true,
+};
+
+const geoAttributesBase = {
+    ...stackBase,
```

Comment:
Do we have `attributesBase`?
If not, we should have own props here for geo attr, instead of depending on `stackBase` which is specific for Highcharts